environment.py

```diff
@@ -84,29 +84,42 @@ def findCellNeighbors(self):
                 self.grid[i][j].findNeighbors()
 
     def findCellsInCardinalRange(self, startX, startY, gridRange):
-        cellsInRange = []
+        cellsInRange = [{"cell": self.grid[startX][startY], "distance": 0}]
```

Comment:
Agents should not consider their current cell as a viable option. Sugarscape assumes agents _must_ move somewhere else with not moving be a fallback if there are truly no other options. Remove this line.

Reply:
Fixed in new commit, but it's worth noting that the currently implemented behavior includes the current cell in `findCellsInRadialRange()`.

Reply:
That should likewise be taken out. Good catch.